test/wsdl-parse-test.js

```diff
@@ -14,4 +15,13 @@ describe(__filename, function () {
       done();
     });
   });
+
+  it('should only parse a WSDL once', function(done) {
```

Comment:
From `require('soap').createClient` point of view, it calls `open_wsdl`, when the WSDL is doing it's imports that's when the recursive WSDLs come into play, proving that a WSDL is only parsed once ( by returning the exact same WSDL object ) we can prove that a WSDL isn't going to set of an infinite loop

I can add a test with recursive WSDLs using `open_wsdl`